book/src/drivers.md

```diff
@@ -39,7 +39,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Hitachi Vantara](https://knowledge.hitachivantara.com/Documents/Adapters_and_Drivers/Storage_Adapters_and_Drivers/Containers) | `com.hitachi.hspc.csi` | v1.0 | A Container Storage Interface (CSI) Driver for VSP series Storage | Persistent | ? | ? | 
 [HPE](https://github.com/hpe-storage/csi-driver) | `csi.hpe.com` | v0.3, v1.0, v1.1 | A Container Storage Interface (CSI) driver from HPE | Persistent and Ephemeral | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
 [IBM Block Storage](https://github.com/ibm/ibm-block-csi-driver) | `block.csi.ibm.com` | v1.0, v1.1 | A Container Storage Interface (CSI) [Driver](https://www.ibm.com/support/knowledgecenter/SSRQ8T) for IBM FlashSystem 9100, IBM Spectrum Virtualize, IBM Storwize, IBM FlashSystem A9000 and A9000R. | Persistent | Read/Write Single Pod | Yes | 
-[IBM Spectrum Scale](https://github.com/IBM/ibm-spectrum-scale-csi) | `spectrumscale.csi.ibm.com` | v1.0, v1.1 | A Container Storage Interface (CSI) [Driver](https://www.ibm.com/support/knowledgecenter/en/STXKQY_5.0.4/com.ibm.spectrum.scale.csi.v5r04.doc/bl1csi_kc_landing.html) for the IBM Spectrum Scale File System | Persistent | Read/Write Multiple Pod | Yes | 
+[IBM Spectrum Scale](https://github.com/IBM/ibm-spectrum-scale-csi) | `spectrumscale.csi.ibm.com` | v1.0, v1.1 | A Container Storage Interface (CSI) [Driver](https://www.ibm.com/support/knowledgecenter/STXKQY_CSI_SHR) for the IBM Spectrum Scale File System | Persistent | Read/Write Multiple Pod | Yes |
```

Comment:
When clicking on the new link "https://www.ibm.com/support/knowledgecenter/STXKQY_CSI_SHR", I got this message: "The requested page does not exist or might have moved."

The original link (https://www.ibm.com/support/knowledgecenter/en/STXKQY_5.0.4/com.ibm.spectrum.scale.csi.v5r04.doc/bl1csi_kc_landing.html) works.

Reply:
There is planned release tomorrow(25/3), so link will be enabled in few hours.

Reply:
The link https://www.ibm.com/support/knowledgecenter/STXKQY_CSI_SHR is now live.

Reply:
Verified the link works now.  Thanks.